### ISSUE TITLE
How to run this code with CUDA >12.0? 

### ISSUE THREAD

**Issue by sev777** [buaacyw/GaussianEditor#70]
simple-knn and diff-gaussian-rasterization could install with cuda 12?

**Comment by RAF3DA** [comment]
Yes,  type torch cpp_extension.py and download it from GitHub. Replace this at C/users/__/.conda/envs/(curr env)/Lib/site-packages/torch/utils

This is for cuda 12.4
